docs/v1/P4Runtime-Spec.mdk

```diff
@@ -3033,10 +3035,10 @@ type: INSERT
 entity {
   packet_replication_engine_entry {
     clone_session_entry {
-      clone_session_id : 100
-      replicas { egress_port : 0xFFFFFFFD replica_id: 1 } # to CPU
+      session_id : 100
+      replicas { egress_port : 0xFFFFFFFD instance: 1 } # to CPU
       class_of_service : 2
-      packet_length_bytes : 4094
+      packet_length_bytes : 4096
```

Comment:
in theory truncating at 4094 is fine, but I respect your love for power of 2s :)

Reply:
I made that change to match the 4096 numbers mentioned text after the code example.  I suppose I could have updated the text to 4094 instead.  Just picked one.

Reply:
sgtm